### PR TITLE
Update nodejs.md

### PR DESCRIPTION
**Pull request by sourabhkatti**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security/application_security/enabling/compatibility/nodejs.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The following operating systems are officially supported by `dd-trace`. Any oper
 
 
 | Framework                | Threat Detection supported? | Threat Protection supported? | Vulnerability Management for Code supported? |
```
Comment (sourabhkatti) [inline content/en/security/application_security/enabling/compatibility/nodejs.md:106]:
```suggestion
| Framework                | Threat Detection supported? | Threat Protection supported? | Vulnerability Management for Code-level supported? |
```

File: content/en/security/application_security/enabling/compatibility/nodejs.md [hunk 2]
```diff
@@ -122,10 +122,10 @@ The following operating systems are officially supported by `dd-trace`. Any oper
 
 ##### ASM Capability Notes
 - **Vulnerability Management for OSS**  is supported on all frameworks
-- **Threat Protection** works at the HTTP request (input) layer, and so works for all databases by default
+- **Threat Protection** also works at the HTTP request (input) layer, and so works for all databases by default, even those not listed in the table below.
 
 
-| Framework                 | Versions | Threat Detection supported? | Threat Protection supported? | Code Vulnerability Detection supported? |
+| Framework                 | Versions | Threat Detection supported? | Threat Protection supported? | Vulnerability Management for Code supported? |
```
Comment (sourabhkatti) [inline content/en/security/application_security/enabling/compatibility/nodejs.md:128]:
```suggestion
| Framework                 | Versions | Threat Detection supported? | Threat Protection supported? | Vulnerability Management for Code-level supported? |
```

**Review by sourabhkatti — COMMENTED** [review]
updating custom code name

**Review by sourabhkatti — COMMENTED** [review]
updating deployment

**Review by aliciascott — APPROVED** [review]

**Pull request merged**